src/Model/ROI.py

```diff
@@ -153,121 +152,128 @@ def create_roi(rtss, roi_name, roi_coordinates, data_set,
-                # if data_set.get("ReferencedImageSequence"):
-                contour.add_new(Tag("ContourImageSequence"), "SQ",
-                                contour_image_sequence)
-
-                # Contour Sequence
-                for contour_image in contour_image_sequence:
-                    contour_image.add_new(
-                        Tag("ReferencedSOPClassUID"), "UI",
-                        referenced_sop_class_uid)  # CT Image Storage
-                    contour_image.add_new(Tag("ReferencedSOPInstanceUID"),
-                                          "UI", referenced_sop_instance_uid)
-
-                contour.add_new(Tag("ContourNumber"), "IS", 1)
-                if not _is_closed_contour(roi_coordinates):
-                    contour.add_new(Tag("ContourGeometricType"), "CS",
-                                    "OPEN_PLANAR")
-                    contour.add_new(Tag("NumberOfContourPoints"), "IS",
-                                    number_of_contour_points)
-                    contour.add_new(Tag("ContourData"), "DS", roi_coordinates)
-                else:
-                    contour.add_new(Tag("ContourGeometricType"), "CS",
-                                    "CLOSED_PLANAR")
-                    contour.add_new(Tag("NumberOfContourPoints"), "IS",
-                                    number_of_contour_points - 1)
-                    contour.add_new(Tag("ContourData"), "DS",
-                                    roi_coordinates[0:-3])
-
-            roi_contour.add_new(Tag("ReferencedROINumber"), "IS", roi_number)
-
-        # Combine original ROIContourSequence with new
-        original_roi_contour.extend(roi_contour_sequence)
-
-        rtss.add_new(Tag("ROIContourSequence"), "SQ", original_roi_contour)
-
-        # Saving a new RTROIObservationsSequence
-        rt_roi_observations_sequence = Sequence([Dataset()])
-
-        original_roi_observation_sequence = rtss.RTROIObservationsSequence
-
-        for ROI_observations in rt_roi_observations_sequence:
-            # TODO: Check to make sure that there aren't multiple observations
-            #  per ROI, e.g. increment from existing Observation Numbers?
-            ROI_observations.add_new(Tag("ObservationNumber"), 'IS',
-                                     roi_number)
-            ROI_observations.add_new(Tag("ReferencedROINumber"), 'IS',
-                                     roi_number)
-            ROI_observations.add_new(Tag("RTROIInterpretedType"), 'CS',
-                                     rt_roi_interpreted_type)
-            ROI_observations.add_new(Tag("ROIInterpreter"), 'CS',
-                                     "")
-
-        original_roi_observation_sequence.extend(rt_roi_observations_sequence)
-        rtss.add_new(Tag("RTROIObservationsSequence"), "SQ",
-                     original_roi_observation_sequence)
+def add_new_roi(rtss, roi_name, roi_coordinates, data_set, rt_roi_interpreted_type):
```

Comment:
```suggestion
def add_new_roi(rtss, roi_name, roi_coordinates, data_set,
                rt_roi_interpreted_type):
```

PEP 8 compliance. Also, need docstrings for this function.